src/Npgsql.SourceGenerators/AnalyzerReleases.Unshipped.md

```diff
@@ -1,4 +1,5 @@
 ### New Rules
 Rule ID | Category | Severity | Notes
 --------|----------|----------|-------
+NPGSQLDBINFOGEN001 | DatabaseInfoSourceGenerator | Error | DatabaseInfoSourceGenerator
```

Comment:
Not sure there's a need for a distinct rule here (e.g. PGXXXX can probably be used). Distinct IDs are useful for user-facing warnings/errors, but this is a class of internal errors that are only interesting to us (and we'd just look at the stack trace). Basically this source generator isn't for public consumption - it's just an internal implementation detail of Npgsql itself.

If you do want a distinct rule, then the convention is to have a two-letter code indicating the source/kind of the message, followed by a number for that rule (that's why below it's PG, could be NP instead).

Reply:
It turns out you can't reuse the same id in different analyzers so we'll need a distinct rule.